Justfile

```diff
@@ -0,0 +1,32 @@
+# Print help message
```

Comment:
What's this?

Reply:
The output of `just help` is auto-generated from comments in the file